design/cli-install-changes.md

```diff
@@ -0,0 +1,286 @@
+        --config mapStringString                          configuration to use for creating a backup storage location. Format is key1=value1,key2=value2 (was also in `velero install --backup-location-config`). Required for Azure.
+        --provider string                                 provider name for backup storage. Required.
+        -h, --help                                        help for create
+        --label-columns stringArray                       a comma-separated list of labels to be displayed as columns
+        --labels mapStringString                          labels to apply to the backup storage location
+        -o, --output string                               Output display format. For create commands, display the object but do not send it to the server. Valid formats are 'table', 'json', and 'yaml'. 'table' is not valid for the install command.
+        --prefix string                                   prefix under which all Velero data should be stored within the bucket. Optional.
+        --provider string                                 name of the backup storage provider (e.g. aws, azure, gcp)
+        --show-labels                                     show labels in the last column
+
+      
+      get                                                 Display backup storage locations
+        --default                                         displays the current default backup storage location (NEW)
+        -h, --help                                        help for get
+        --label-columns stringArray                       a comma-separated list of labels to be displayed as columns
+        -o, --output string                               Output display format. For create commands, display the object but do not send it to the server. Valid formats are 'table', 'json', and 'yaml'. 'table' is not valid for the install command. (default "table")
+        -l, --selector string                             only show items matching this label selector
+        --show-labels                                     show labels in the last column
+
+```
+
+3) `velero snapshot-location`
+Commands/flags for snapshot locations. 
+
+```
+     set 
+        --default mapStringString                         sets the list of unique volume providers and default volume snapshot location (provider1:location-01,provider2:location-02,...) (NEW, -- was `server --efault-volume-snapshot-locations)  
+
+      create
+        --dry-run                                         generate resources, but don't send them to the cluster. Use with -o. Optional. (NEW)
+        --default                                         Sets these new locations to be the new default snapshot locations. Default is false. (NEW) 
+        --secret-file string                              file containing credentials for volume provider. If not specified, set --no-secret must be used for confirmation. Optional. (MOVED FROM install)
```

Comment:
Echoing Steve's comment, I think the `--secret-file` flag probably goes better with `velero plugin add` since it's related to the plugin. We can't really do secrets per location right now.